src/Raven.Server/Documents/DocumentFlags.cs

```diff
@@ -56,7 +56,8 @@ public enum NonPersistentDocumentFlags
         ByTimeSeriesUpdate = 0x4000,
         LegacyDeleteMarker = 0x8000,
         ForceRevisionCreation = 0x10000,
-        AllowDataAsNull = 0x20000
+        AllowDataAsNull = 0x20000,
+        FromImport = 0x40000
```

Comment:
what is the difference between this and 'FromSmuggler' flag?

Reply:
From my understanding, in places where the `FromSmuggler` is used, we expect a revision to be created (in most cases).
We can see here clearly that there are cases that revision should by created when we have this flag:
![image](https://github.com/user-attachments/assets/f0327c33-84cc-450d-a2d6-3f9f27c9b918)